submissions/krylenger/html-movie-seat-booking/app.js

```diff
@@ -0,0 +1,122 @@
+const bodyContainer = document.getElementById('body-container');
+const hall = document.getElementById('choose-section__hall');
+const confirmationSection = document.getElementById('choose-section__chosen-tickets');
+const confirmationSectionInnerContainer = document.getElementById('chosen-tickets__inner');
+const ticketsContainer = document.getElementById('chosen-tickets__tickets-container');
+const checkoutButton = document.getElementById('checkout-seats__checkout-button');
+
+const state = {
+  chosenSeats: [],
+};
+
+const removeSeat = (currentSeat, chosenSeatsArray) =>
+  (chosenSeatsArray = chosenSeatsArray.filter((seatInArray) => seatInArray !== currentSeat));
+
+const addSeat = (seat, chosenSeatsArray) => {
+  chosenSeatsArray.push(seat);
+  return chosenSeatsArray;
+};
+
+const checkIfSeatAlreadyChosen = (seat, chosenSeatsArray) =>
+  (chosenSeatsArray = chosenSeatsArray.includes(seat)
+    ? removeSeat(seat, chosenSeatsArray)
+    : addSeat(seat, chosenSeatsArray));
+
+const updateChosenSeats = (seat, chosenSeatsArray) =>
+  (chosenSeatsArray = checkIfSeatAlreadyChosen(seat, chosenSeatsArray));
+
+createTicketTemplate = ([row, seat], index) => `<div class="chosen-tickets__ticket">
+  <span class="chosen-tickets__dot"></span>
+  <span class="chosen-tickets__span-bold">${++index}.</span>
+  <div class="chosen-tickets__row-seat">
+    <span>Row: ${row}</span>
+    <span>Seat: ${seat}</span>
+  </div>
+  <span class="chosen-tickets__span-bold">7$</span>
+</div>`;
+
+createModalConfirmationTemplate = () => `<section class="choose-section__chosen-tickets choose-section__chosen-tickets--modal chosen-tickets__modal-wrapper" id="choose-section__chosen-tickets">
+<div class="chosen-tickets chosen-tickets__inner" id="chosen-tickets__inner">
+  <div class="chosen-tickets__header">
+    <span class="chosen-tickets__dot chosen-tickets__dot--main"></span>
+    <button class="chosen-tickets__button--go-back" id="chosen-tickets__button--go-back">cancel</button>
+  </div>
+  <h3 class="choose-section__title">Please double check</h3>
+  <div class="chosen-tickets__tickets-container chosen-tickets__tickets-container--modal" id="chosen-tickets__tickets-container--modal">
+  </div>
+  <a class="checkout-seats__checkout-button" id="checkout-seats__checkout-button--modal" type="submit">
```

Comment:
Submit button not working. Please fix it
![image](https://user-images.githubusercontent.com/16196199/109871884-5475f800-7c74-11eb-84bb-9c04f5cb76e7.png)

Reply:
I've fixed this.
Now it renders the success message, closes modal window after a short delay and resets the initial application.
Could you please check briefly the structure of my app and tell is it proper structured or maybe it should be done in other (more efficient) way. 
Thank you in advance!